### DISCUSSION TITLE
Outer glow effect with any color

### DISCUSSION THREAD

**Discussion by Fekkesh** [pmndrs/postprocessing#362] [Q&A]
Hello!

I'm trying to add a black outer glow to a white TextGeometry so it will be more readable against the background.
I'm also trying to add a blue outer glow to a black ShapeBufferGeometry. I tried to use BloomEffect but it seems to glow in the same color as the object. Is there a way to change the glow color? 
I don't want the object to be affected by the effect, for example if I set the objects color to 0xff0000, it must remain that color after the effect is applied, I only want to add an outer glow, in a different color.  
What effect must I use to achieve this?

**Comment by vanruesc** [comment]
You could try the `OutlineEffect` instead. It should be possible to create a black glow with that by using a black edge color with a large `blurKernel` and `BlendFunction.ALPHA`.

**Reply by Fekkesh** [reply]
Allright, thanks! I will try that.
